repo/packages/K/koding/0/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "2.0",
+  "name": "koding",
+  "version": "0.0.1",
```

Comment:
`version` should be "kodingVersion-universePackageVersion". Ex: if koding's version for singular is 1.2.3 then `version` should be "1.2.3-0.0.1"